lib/fastlane/plugin/mango/helper/mango_helper.rb

```diff
@@ -24,6 +25,8 @@ def initialize(params)
         @pre_action = params[:pre_action]
         @docker_registry_login = params[:docker_registry_login]
         @pull_latest_image = params[:pull_latest_image]
+
+        @docker_commander = DockerCommander.new
```

Comment:
If we make `DockerCommander` a module, we may include it ant use methods without initializing or calling the class.